components/patient-data/indexing/src/test/java/org/phenotips/data/indexing/internal/SolrPatientIndexerTest.java

```diff
@@ -91,8 +91,8 @@
 {
     private static final String STATUS_KEY = "status";
 
-    private static final List<String> STATUS_VALUES = Arrays.asList("candidate", "rejected", "rejected_candidate",
-        "solved", "carrier");
+    private static final List<String> STATUS_VALUES = Arrays.asList("candidate", "rejected", "rejectedcandidate",
+        "solved", "carrier", "candidate_novel_disease", "candidate_novel_phen", "umc", "umc_vus", "umc_msv");
```

Comment:
These values should be extracted from the XClass instead of being duplicated here